app/backend/src/couchers/phone/sms.py

```diff
@@ -25,11 +29,13 @@ def send_sms(number, message):
     success, "unsupported operator" on unsupported operator, and any other
     string for any other error."""
 
-    assert len(message) <= 140, "message too long"
+    assert len(message) <= 140, "Message too long"
 
     if not config["ENABLE_SMS"]:
         logger.info(f"SMS not enabled, need to send to {number}: {message}")
-        return
+        if config["DEV"]:
+            return "success"
+        return "SMS not enabled."
```

Comment:
You're testing one config variable too many here. Why do you want to return success when no sms was sent?

Reply:
So that it's possible to test it by clicking around locally. Otherwise all the api calls fail.

Reply:
ok, fair enough. The testconfig autofixture sets DEV so I'm not sure why your added test works... will look into it.

Reply:
aha, test didn't work :) Will have another look after you test run before commit :D